src/material/core/tokens/_m3-tokens.scss

```diff
@@ -170,7 +170,8 @@
     }
     @else if($color != null) {
       $result: map.remove($result, $opacity-key);
-      $result: map.set($result, $color-key, #{color(from #{$color} srgb r g b / #{$opacity})});
+      $combined-color: #{color(from #{$color} srgb r g b / #{$opacity})};
```

Comment:
I realized we can write this slightly more compactly as `rgb(from ... r g b / 1)` https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_colors/Relative_colors#color_functions_that_support_relative_colors

Reply:
I had to write it as `@rgb(from $color #{r g b} / map.get($args, alpha));` to get sass to accept it. Otherwise I get an error:
```
sass.Exception [Error]: Only 3 elements allowed, but 5 were passed.
    ╷
173 │       $combined-color: rgb(from $color r g b / $opacity);
    │                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ╵
```

Does that seem ok?

Reply:
actually, weren't we going to change this whole thing to color-mix because of firefox support? (but yeah that looks good for future when firefox supports it)

Reply:
Oh shoot, yeah. I forgot about that. I pushed up the color-mix solution